omc3/model/accelerators/accelerator.py

```diff
@@ -285,3 +320,23 @@ class AcceleratorDefinitionError(Exception):
     should have been overwritten.
     """
     pass
+
+
+# Helper ----
+
+def _get_modifiers_from_modeldir(model_dir: Path):
+    """Parse modifiers from job.create_model.madx or use modifiers.madx file."""
+    job_file = model_dir / JOB_MODEL_MADX
+    if job_file.exists():
+        job_madx = job_file.read_text()
+        modifiers = re.findall(fr"\s+call,\s*file\s*=\s*[\"\']?([^;\'\"]+)[\"\']?\s*;\s*{MODIFIER_TAG}", job_madx,
```

Comment:
Could use a comment to explain that regex, so we don't need to think about it too much later on